{
  "PublicDateAtUSN": "0001-01-01T00:00:00Z",
  "CRD": "0001-01-01T00:00:00Z",
  "Candidate": "CVE-2024-9774",
  "PublicDate": "2024-12-27T02:15:00Z",
  "References": [
    "https://www.cve.org/CVERecord?id=CVE-2024-9774",
    "https://discuss.tryton.org/t/security-release-for-issue-93"
  ],
  "Description": "A vulnerability was found in python-sql where unary operators do not escape non-Expression.",
  "UbuntuDescription": "",
  "Notes": null,
  "Bugs": [
    "https://foss.heptapod.net/tryton/python-sql/-/issues/93"
  ],
  "Priority": "medium",
  "DiscoveredBy": "CÃ©dric Krier",
  "AssignedTo": "",
  "Patches": {
    "python-sql": {
      "devel": {
        "Status": "needs-triage",
        "Note": ""
      },
      "esm-apps/bionic": {
        "Status": "needs-triage",
        "Note": ""
      },
      "esm-apps/focal": {
        "Status": "needs-triage",
        "Note": ""
      },
      "esm-apps/jammy": {
        "Status": "needs-triage",
        "Note": ""
      },
      "esm-apps/noble": {
        "Status": "needs-triage",
        "Note": ""
      },
      "esm-apps/xenial": {
        "Status": "needs-triage",
        "Note": ""
      },
      "focal": {
        "Status": "needs-triage",
        "Note": ""
      },
      "jammy": {
        "Status": "needs-triage",
        "Note": ""
      },
      "noble": {
        "Status": "needs-triage",
        "Note": ""
      },
      "oracular": {
        "Status": "needs-triage",
        "Note": ""
      },
      "upstream": {
        "Status": "released",
        "Note": "1.5.2-1"
      }
    }
  },
  "UpstreamLinks": {
    "python-sql": [
      "https://foss.heptapod.net/tryton/python-sql/-/commit/f20551bbb8b3b4c4dd0a2c3d36f377bff6f2f349"
    ]
  }
}